controls/radspreadsheet/features/ui-headers-and-footers.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Headers and Footers
+page_title: Headers and Footers
+description: Headers and Footers
+slug: radspreadsheet-ui-headers-and-footers
+tags: header,footer
+published: True
+position: 6
+---
+
+# Headers and Footers
+
+
+Headers and Footers functionality allows you add rich text content in the page margins when exporting a worksheet to pages. This feature is useful in scenarios like [exporting to PDF](http://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/formats-and-conversion/pdf/pdfformatprovider), [printing]({%slug radspreadsheet-ui-printing%}) or [import/export to XLSX](http://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/formats-and-conversion/xlsx/xlsxformatprovider). 
+
+
+## Overview
+
+RadSpreadsheet allows you add headers and/or footers at the top or bottom of a worksheet. The applied headers and footers are not displayed on the worksheet when the document is visualized in an application — they appear on the printed pages and in the exported PDF document only. 
```

Comment:
Consider inverting the second sentence and move the positive part in the beginning.